tests/general/testdata/internal_metrics_expected.yaml

```diff
@@ -35,15 +32,6 @@ resourceMetrics:
                     - key: exporter
                       value:
                         stringValue: otlp
-                    - key: service.instance.id
```

Comment:
note - these were duplicated as resource and datapoint attributes. Now they are only in resource attr.